browser/help/topics/constraint.md

```diff
@@ -62,7 +62,7 @@ As mentioned above, `oe` and `LOEUF` are dependent on sample size and we note th
 
 For synonymous and missense variation, we created a signed Z score of the deviation of observed counts from the expected number. Positive Z scores indicate increased constraint (intolerance to variation) and therefore that the transcript had fewer variants than expected. Negative Z scores were given to transcripts that had more variants than expected.
 
-To generate Z scores, we used a previously described, but slightly modified, sequence-context based mutational model to predict the number of expected rare (minor allele frequency < 0.1%) variants per transcript at well covered sites. We then calculated the chi-squared value for the deviation of observation from expectation for each mutational class (synonymous and missense). The square root of these values was taken and multiplied by -1 if the number of observed variants was greater than expectation or 1 if observed counts were smaller than expected. The synonymous Z scores were then corrected by dividing each score by the standard deviation of all synonymous Z scores in between -5 and 5. For the missense Z scores, we took all Z scores between -5 and 0 and created a mirrored distribution. The missense Z scores were then corrected by dividing each score by the standard deviation of these mirror distributions.
+To generate Z scores, we used a previously described, but slightly modified, sequence-context based mutational model to predict the number of expected rare (minor allele frequency < 0.1%) variants per transcript at well covered sites. We then calculated the chi-squared value for the deviation of observation from expectation for each mutational class (synonymous and missense). The square root of these values was taken and multiplied by -1 if the number of observed variants was greater than expectation or 1 if observed counts were smaller than expected. The synonymous Z scores were then corrected by dividing each score by the standard deviation of all synonymous Z scores in between -8 and 8 (-5 and 5 for gnomAD v2). For the missense Z scores, we took all Z scores between -5 and 0 (-5 and 0 for gnomAD v2) and created a mirrored distribution. The missense Z scores were then corrected by dividing each score by the standard deviation of these mirror distributions.
```

Comment:
minor comment for this:
>For the missense Z scores, we took all Z scores between -5 and 0 (-5 and 0 for gnomAD v2)

should we reword to something like: "For the missense Z scores, we took all Z scores between -5 and 0 (same cutoffs used in gnomAD v2) ..."

Reply:
meant to change that to a -8

Reply:
oh whew I wasn't sure if we only updated LoF